files/default/handler/audit_report.rb

```diff
@@ -38,6 +38,11 @@ def report
         # load inspec, supermarket bundle and compliance bundle
         load_needed_dependencies
 
+        # check if we have a valid version for backend caching
+        if node['audit']['inspec_backend_cache'] && (Gem::Version.new(::Inspec::VERSION) < Gem::Version.new('1.47.0'))
```

Comment:
We're now checking for a specific version constraint in two different places within this file... here, and:

https://github.com/chef-cookbooks/audit/blob/master/files/default/handler/audit_report.rb#L134-L137

I think we should move the MIN_INSPEC_VERSION one to be right above this, and out of the `#call` method. Or, possibly better yet, extract both to a `validate_inspec_version!` method and call it here in the `#report` method in place of this stanza. What do you think?